### DISCUSSION TITLE
Editor freezes when removing text or pressing enter on larger files

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#38996] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Whenever I edit a larger file (in this case a relatively large Markdown file) and either remove text (through selecting and pressing backspace) or press enter is the editor freezing for several seconds before updating.

Small video showing this. I pressed backspace right after selecting the text (At around 4 seconds).

https://user-images.githubusercontent.com/11576465/201525981-c04b9db9-e066-4d39-aca9-d39316cdbf00.mp4

**Comment by jfullstackdev** [comment]
I see that's Firefox, there are issues of lagging in Firefox I'm seeing recently here, even in Codespaces

try Chrome, if there is none, then it's on Firefox side

unfortunately, most websites are compatible with Chrome, some practices are not in Firefox and there is an issue regarding inefficient use of RAM by Firefox, unlike Chrome, though Chrome is hogging the RAM, it's effecient
